TypeFinder.py

```diff
@@ -74,6 +74,45 @@ def __init__(self, file_name):
         self.norm_flux = norm_flux
 
 
+class Templates:
+
```

Comment:
Good use of modules to reduce redundancy in plot code and good use of classes to add diversity to the types of plots.